templates/pages/members.html

```diff
@@ -4,21 +4,50 @@
 
 {% block content%}
 
-
-	<div>
-		<h1>All CodeCatz Members</h1>
-		<p>This page will show all our members and who is working on what.</p>
-		<p>Find below example for one member</p>
+	<header class="row">
+		<div class="col-lg-6 col-sm-7">
+			<h1>Who is who?</h1>
+		</div>
+		<div class="col-lg-6 col-sm-5">
+			<p>This page will tell a bit more about CodeCatz members.</p>
+		</div>
+	</header>
+	<div class="row">
+		<div class="col-md-3">
+			<p><img src="http://placekitten.com/g/350/350" alt="" class="img-responsive"></p>
+		</div>
+		<div class="col-md-10">
+			<h4>Little kitten 1 (name)</h4>
+			  <p>About Kitten 1:Fun, ragdoll Gizmo boots feel caturday home whisker. Kitten, family run buddy curiosity furball sleeping boots. Backyard kitten friend's long pillow cat sunbathe beautiful petting kittens siam tabby dog baby lady cute disapproval belly her. Siam spoon buddy house smile feed me this spoon home cats fire cat meet.</p>
```

Comment:
there are unneeded spaces in this line (same goes for line 39)...
never mind for now, just as a note for the future :wink: